data/json/furniture_and_terrain/furniture-tools.json

```
@@ -1638,6 +1638,37 @@
       ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_vending_reinforced_off",
+    "name": "reinforced vending machine (off)",
+    "looks_like": "f_vending_reinforced",
+    "description": "A bit tougher to crack open than a regular vending machine.  Out of order, for lack of power.  That just makes it all the sweeter of a target, doesn't it?",
+    "symbol": "{",
+    "color": "light_red",
+    "move_cost_mod": -1,
+    "coverage": 90,
+    "required_str": 30,
+    "flags": [ "SEALED", "PLACE_ITEM", "CONTAINER", "BLOCKSDOOR", "FLAMMABLE_HARD", "MINEABLE" ],
```

```suggestion
    "flags": [ "SEALED", "PLACE_ITEM", "CONTAINER", "BLOCKSDOOR", "MINEABLE" ],
```
Don't see how it'd be flammable?

---

it is a direct copy of active reinforced vending machine, i guess i'll edit their flags also